### ISSUE TITLE
Select a code and see all instances of that code (e.g., list of open-ended questions)

### ISSUE THREAD

**Issue by neilthawani** [neilthawani/classinsight-prototype#21]

**Comment by neilthawani** [comment]
Alternatively, we could ideate ways to create the app as a multi-page dashboard. It's already starting to look like this because of how we are dividing coarse, medium, and fine-grained visualizations. Right now there are three buttons for talk ratio, turn taking, and transcript.

We could define this as a "fine-grained" visualization.

We could also have this "code instance" visualization overlay on each (talk ratio, turn taking, transcript) visualization.

**Comment by angelaebstewart** [comment]
I agree to filter by selecting codes and seeing all the transcripts that correspond to that code. They will also likely need to be contextualized. So I think there should be a way to see (or scroll) the immediately preceding and succeeding discourse. To do this multi-page (not sure that's necessary), perhaps have "view tabs" within the transcript visualization page.

**Comment by AndiGomoll** [comment]
I was imagining that within the fine grain view, teachers would be able to view the current highlight functionality across the whole lesson, and then they might also be able to push a button beside the code that says "show list" (or something like this) to toggle to a list of all questions that fit that code. Then, as Angela is describing, they could click on specific questions within this list to see some context. This may be too complex for the current build... what do you think?

**Comment by neilthawani** [comment]
> I agree to filter by selecting codes and seeing all the transcripts that correspond to that code. They will also likely need to be contextualized. So I think there should be a way to see (or scroll) the immediately preceding and succeeding discourse. To do this multi-page (not sure that's necessary), perhaps have "view tabs" within the transcript visualization page.

Currently, the Turn Taking visualization has a "drill-down" function. If you click on a bar, it shows the preceding and succeeding turn to the one clicked. I can add an additional function:
Click the Legend on TurnTaking to toggle visible utterance types (https://github.com/neilthawani/classinsight-prototype/issues/212)

**Comment by neilthawani** [comment]
> I was imagining that within the fine grain view, teachers would be able to view the current highlight functionality across the whole lesson, and then they might also be able to push a button beside the code that says "show list" (or something like this) to toggle to a list of all questions that fit that code. Then, as Angela is describing, they could click on specific questions within this list to see some context. This may be too complex for the current build... what do you think?

I think we can toggle all questions that pertain to a code from the TalkRatio visualization (https://github.com/neilthawani/classinsight-prototype/issues/213). We should discuss how users might hop between the three visualizations tomorrow, too. Context is very important.
